### PR TITLE
docs/release team provisioning: Add members for v1.22 Release Docs.

### PR DESCRIPTION
**Pull request by PI-Victor**
Add users for v1.22 Release team as part of the sig-release team and
website-milestone-maintainers (gives them access to set /milestone
for PRs to k/website as part of their workflow for RT).

Signed-off-by: Victor Palade <victor@cloudflavor.io>

/assign @savitharaghunathan 
~/hold for [@ritpanjw access to the org to be resolved](https://github.com/kubernetes/org/issues/2665#issuecomment-828222961)~

### CODE REVIEW

File: config/kubernetes/sig-release/teams.yaml
Comment (palnabarun) [inline config/kubernetes/sig-release/teams.yaml:259]:
```suggestion
        - chris-short # 1.20 Communications Shadow
        - chrisnegus # 1.22 Release Docs Shadow
```

**Review by palnabarun — CHANGES_REQUESTED** [review]
small nit

**Comment by PI-Victor** [comment]
@palnabarun thank you, but @ritpanjw first needs to accept the invite into the org, i think it was sent out today, i saw you took care of the Issue, will remove the hold once she has done so!

File: config/kubernetes/sig-release/teams.yaml
Comment (PI-Victor) [reply config/kubernetes/sig-release/teams.yaml:259]: ah, will fix, thanks!

**Comment by palnabarun** [comment]
@PI-Victor -- I see that @ritpanjw has already accepted the membership invite. You are good to remove the hold now. :)

**Comment by palnabarun** [comment]
/lgtm

leaving approval for @kubernetes/sig-release-leads

**Review by hasheddan — APPROVED** [review]
/lgtm

**Comment by PI-Victor** [comment]
branch was out-of-sync and resulted in a merge conflict on force push, fixed now. please lgtm and approve when you can, thanks!

**Comment by PI-Victor** [comment]
also needs an approval from @kubernetes/sig-docs-leads since it adds the shadows to the team

File: config/kubernetes/sig-docs/teams.yaml
Comment (kbhawkey) [inline config/kubernetes/sig-docs/teams.yaml:359]: Hi. The changes look good. How is this list managed (pruned)?

Comment (PI-Victor) [reply config/kubernetes/sig-docs/teams.yaml:359]: @kbhawkey good point, that's on my list to discuss on the next docs meeting. This and `website-milestone-maintainers`.

**Pull request merged**
